### ISSUE TITLE
Do not use zoom for lock screen wallpaper

### ISSUE THREAD

**Issue by evitalis** [mate-desktop/mate-screensaver#168]
#### Expected behaviour
Lock screen wallpaper matches the user's actual wallpaper and size settings. This issue complements #55 and #131 

#### Actual behaviour
Lock screen wallpaper uses the zoom option which causes it to display distorted.

#### Steps to reproduce the behaviour
Simply locking the screen and bringing up the login dialog will reproduce this easily and quickly.

#### MATE general version
1.20

#### Package version
1.20

#### Linux Distribution
Fedora 28, OpenBSD-current

#### Link to downstream report of your Distribution
No links as mate-screensaver is the upstream where the issue exists.

**Comment by evitalis** [comment]
I will have to get the versions from my Fedora system but on OpenBSD I see the following.

```
[19:46:29] USER@HOSTNAME:~ $ pkg_info mate-screensaver
Information for inst:mate-screensaver-1.20.0p2

Comment:
MATE screensaver

Required by:
mate-panel-1.20.1p2

Description:
Mate-screensaver is MATE-based screen saver and locker that aims to have simple,
sane, secure defaults and be well integrated with the desktop.

Maintainer: Robert Nagy <robert@openbsd.org>

WWW: http://mate-desktop.org/
```

My Fedora workstation is a simple F28 install with all updates applied. Imagine if you set the wallpaper using the 'zoom' option, that is what happens on my systems when I lock the PC. The wallpaper on the lock screen is simply distorted.

**Comment by raveit65** [comment]
> The wallpaper on the lock screen is simply distorted.

Screenshot or a photo?
Latest stable mate-screenshot version is 1.20.1 in general.
Is bsd maintainer sleeping?
fedora 1.20.1-1 and in updates-testing is 1.20.1.2.

**Comment by monsta** [comment]
Is it with one monitor or more, and what is the resolution?

**Comment by evitalis** [comment]
More than one monitor, and the normal resolution is 1680x1050. I think a recent update may have helped as I did not notice the issue last time I had a locked system. Will double check before closing the issue.

**Comment by monsta** [comment]
Did you get an update to 1.20.2?

**Comment by evitalis** [comment]
Hello @monsta sorry for the delays. I did get an update on OpenBSD and Fedora and have not noticed this issue persist. Thank you

**Comment by monsta** [comment]
Thanks, good to know. I guess this means https://github.com/mate-desktop/mate-screensaver/issues/131 is resolved by 1.20.2 as well.
